lms/static/sass/views/_track_selection.scss

```diff
@@ -163,6 +163,8 @@ a:not(.btn), a:visited:not(.btn) {
     color: #00688d !important;
     font-weight: 700;
     font-size: 14px !important;
+    display: none;
+    text-decoration: underline;
 }
```

Comment:
Above CSS should not have been cleaned up in my previous PR, so it broke "show more". Adding the fix back in.